src/api/app/models/maintenance_incident.rb

```diff
@@ -5,6 +5,44 @@ class MaintenanceIncident < ActiveRecord::Base
   belongs_to :project, class_name: "Project", foreign_key: :db_project_id
   belongs_to :maintenance_db_project, :class_name => "Project"
 
+  def self.build_maintenance_incident(project, options = {})
```

Comment:
Because sometimes I call

```
build_maintenance_incident(project, request: my_request)
```

and sometimes


```
build_maintenance_incident(project, no_access: my_access)
```

Otherwise it'd be necessary to call the function with ```nil``` value and that would be ugly (at least I think so ...)